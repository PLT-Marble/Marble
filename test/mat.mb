main(){
    matrix a;
    int b;
    b = 1;
    a = [1.1,2.2;3.3,4.4];
    //a = [1,2;3,4];
}