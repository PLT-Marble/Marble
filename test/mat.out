scanner find id: 
main
scanner find id: 
matrix
scanner find id: 
a
scanner find id: 
int
scanner find id: 
b
scanner find id: 
b
scanner find id: 
a


inside main: 
VDeclare: matrix a;

VDeclare: int b;

Assign: b = <int>1;

Assign: a = <float>1.1, <float>2.2
<float>3.3, <float>4.4;
